lib/rules/no-pause-test.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const emberUtils = require('../utils/ember');
+const types = require('../utils/types');
+
+const ERROR_MESSAGE = 'Do not use `pauseTest()`';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Disallow usage of pauseTest() helper in tests',
```

Comment:
Can you put backticks around the function name `pauseTest()` and then run `yarn update`? This way it will show up as code in the README.